.github/dependabot.yml

```diff
@@ -13,16 +13,6 @@ updates:
   ignore:
     - dependency-name: "notebook"
       versions: '>= 7.0.0'
-- package-ecosystem: pip
-  directory: "/docker_image_dependencies"
-  schedule:
-    interval: monthly
-    time: '11:00'
-  groups:
-    docker-image-python-deps:
-      patterns:
-        "*"
-  open-pull-requests-limit: 20
```

Comment:
all the requirements files are now in the /src directory, so 🤞 dependabot will be a bit more dependable in reporting updates